### DISCUSSION TITLE
Transmitter Status: Unknown error

### DISCUSSION THREAD

**Discussion by lbarmen** [NightscoutFoundation/xDrip#2168] [Q&A]
Hello! Please tell me why the Transmitter Status: Unknown error? Prevents my Dexcom G6 from starting in native mode.
Battary new
Samsung s10, android 12
![Screenshot_20220608-120706_xDrip+](https://user-images.githubusercontent.com/22907206/172625850-b8fa9ecf-e6c5-4376-b230-c9cc8943d780.jpg)
![Screenshot_20220608-115907_xDrip+](https://user-images.githubusercontent.com/22907206/172625855-9ff80705-b4f3-48e8-a1af-d89ed0f44954.jpg)

**Comment by Navid200** [comment]
What exactly happened including what happened before this start?
Did you have a running session on this same transmitter after the battery change and then you restarted?
Or, is this the first timer you are trying to use this transmitter after the battery change?

Please provide the detail and the sequence of events.
My concern is that if you have a session running and then it stops and you want to restart, you cannot do that without removing the transmitter from the sensor.

Another concern I have is that you have engineering mode enabled.
You are free to enable it.  But, why?
There are so many more things that can be causing a problem with engineering mode enabled.

**Reply by lbarmen** [reply]
![Screenshot_20220608-181802_xDrip+](https://user-images.githubusercontent.com/22907206/172654113-02c0968d-6035-41d2-bc1a-0605e37eb3fd.jpg)
![Screenshot_20220608-181751_xDrip+](https://user-images.githubusercontent.com/22907206/172654120-216bff0f-ac5c-4026-b9f8-dbb603df7cc2.jpg)
 
![Screenshot_20220608-181745_xDrip+](https://user-images.githubusercontent.com/22907206/172654123-8c7ee650-0f72-4827-93a9-868fc5344e1d.jpg)
that's what it shows now, though before taking the screenshots turned off the engineering mode, check if it helps

**Reply by Navid200** [reply]
I don't speak your language.  What does it say for "Last Connected"?
Can you translate to English please?

**Reply by lbarmen** [reply]
I don't quite understand what I need to do. in xdrip enable english?

**Reply by Navid200** [reply]
No, just tell me what it means.  Is it 4 minutes?
Is last connected less than 5 minutes or is it more than 5 minutes?
Can you not answer that?

**Reply by Navid200** [reply]
Is this 4 minutes ago, or what?
![172654120-216bff0f-ac5c-4026-b9f8-dbb603df7cc2](https://user-images.githubusercontent.com/51497406/172657331-80a27201-ff8c-4d07-85e4-2d158f8d0dd9.jpg)

**Reply by lbarmen** [reply]
I hope I understood you.

In the screenshot, the last connection time was 4 minutes ago, I received my sugar data in G5 mode

The connection to the transmitter is stable, I can receive data from there in G5 mode and watch sugar.

**Reply by Navid200** [reply]
Perfect.

In native mode, the transmitter is in charge and will not let you restart unless you enable preemptive restart.

You should be able to perform a restart, as explained in the linked document, to get it going in native mode:
https://navid200.github.io/xDrip/docs/Restart-G6-sensor.html
Please pay attention to the requirement to calibrate carefully to address the impact of restart on accuracy.

**Comment by lbarmen** [comment]
Story:
I already have a dexcom g6 transmitter in which I change batteries, there is no such error and I use it in native mode. I bought another transmitter which was already opened.

I inserted a battery into the new transmitter and immediately saw the Transmitter Status: Unknown error, but did not pay attention to it.

Next, I installed a new sensor and transmitter. I found that he did not want to run the sensor and turned off the native mode. It works without native mode.

- I needed the engineering mode to work in the mode G5
- I have not used this transmitter before, it is new
- I was doing hard reset transmitter

I'm sure that the whole problem is in Transmitter Status: Unknown error , this status is shown even in G5 mode

Please help

**Reply by Navid200** [reply]
Transmitter Status: Unknown is an effect.  It is not a cause.
Something is causing it.  There are different situations that could result in connectivity issues.  Many connectivity issues will result in that message showing up.

What does it show now for "Last Connected"?
Is it less than 5 minutes?

**Comment by lbarmen** [comment]
After disabling engineering mode:
![Screenshot_20220608-182223_One UI Home](https://user-images.githubusercontent.com/22907206/172655465-bd370468-9a39-42c2-95e1-2c5b83484c02.jpg)
![Screenshot_20220608-182240_xDrip+](https://user-images.githubusercontent.com/22907206/172655475-e5524b05-edb6-4c98-8285-fb88acc290ae.jpg)
![Screenshot_20220608-182250_xDrip+](https://user-images.githubusercontent.com/22907206/172655479-f6721edb-845d-4859-8cbb-c17531962713.jpg)
